### ISSUE TITLE
Frame Timing Feature Request

### ISSUE THREAD

**Issue by GoldenScream** [piskelapp/piskel#991]
Hey, I just wanted to propose a feature that quite a few professional pixel art animation software has. The ability to choose how long a certain frame will last in the animation, rather than duplicating a frame a lot and hoping it will work with the timing. I could be missing it as well, so let me know if that's the case

**Comment by OPStellar** [comment]
I agree, this would be a useful feature to have. Though I'm not sure if it's realistic for Piskel to do, it is worth investigating the possibility of adding this feature.
